### ISSUE TITLE
Layer type 

### ISSUE THREAD

**Issue by EsiNaderi** [tianzhi0549/CTPN#98]
In the configuration file, the last layer type is "Python", and I get this this error while running demo:

I0531 23:03:25.351613  2315 layer_factory.hpp:76] Creating layer proposal
F0531 23:03:25.351629  2315 layer_factory.hpp:80] Check failed: registry.count(type) == 1 (0 vs. 1) Unknown layer type: Python (known types: AbsVal, Accuracy, ArgMax, BNLL, BatchNorm, Concat, ContrastiveLoss, Convolution, Crop, Data, Deconvolution, Dropout, DummyData, Eltwise, Embed, EuclideanLoss, Exp, Filter, Flatten, HDF5Data, HDF5Output, HingeLoss, Im2col, ImageData, InfogainLoss, InnerProduct, LRN, Log, Lstm, MVN, MemoryData, MultinomialLogisticLoss, PReLU, Pooling, Power, ROIPooling, ReLU, Reduction, Reshape, Reverse, SPP, Sigmoid, SigmoidCrossEntropyLoss, Silence, Slice, SmoothL1Loss, Softmax, SoftmaxWithLoss, Split, TanH, Threshold, Tile, Transpose, WindowData)
*** Check failure stack trace: ***
Aborted (core dumped)

How can I resolve this issue?

**Comment by vinayakarannil** [comment]
Getting the same issue
